### DISCUSSION TITLE
Meeting Minutes Template

### DISCUSSION THREAD

**Discussion by tmorshed** [NeuroTechX/EEG-ExPy#244] [Discussion and Issue Templates]
To be used for Developer Meetings. Based on #243. See the next item for this

- Name your new discussion as **Meeting Minutes DD-MM-YYYY**

To use:
- From the top left, go to "edit"
- Copy the whole markdown document
- Paste in your new issue

**Comment by tmorshed** [comment]
# Meeting Minutes
## Meeting Information
**Meeting Date/Time:** Date in ISO format: YYYY-DD-MM
**Attendees:**  @ tag the attendees, preferably.
**Note Taker:**  

## Agenda Items

Item | Description
---- | ----
Updates | X min
Issues Review | X min 
Other task | X min

## Discussion Items
Item | Who | Notes | Attritbuted time |
---- | ---- | ---- | --- |
what | @ who | when | X mins

## Action Items
Item | Responsible | Due Date |
---- | ---- | ---- |
what | who | when

## Other Notes & Information
- Free text details
- Naming convention for the title of the discussion in use: **"Meeting Minutes DD-MM-YYYY"**


<break>

The note above in raw Markdown:

```markdown
# Meeting Minutes
## Meeting Information
**Meeting Date/Time:** Date in ISO format: YYYY-DD-MM
**Attendees:**  @ tag the attendees, preferably.
**Note Taker:**  

## Agenda Items

Item | Description
---- | ----
Updates | X min
Issues Review | X min 
Other task | X min

## Discussion Items
Item | Who | Notes | Attritbuted time |
---- | ---- | ---- | --- |
what | @ who | when | X mins

## Action Items
Item | Responsible | Due Date |
---- | ---- | ---- |
what | who | when

## Other Notes & Information
- Free text details
- Naming convention for the title of the discussion in use: **"Meeting Minutes DD-MM-YYYY"**
```

**Reply by pellet** [reply]
@tmorshed I'm having trouble getting at the markdown using the edit button, it will only show the result formatted text.  Maybe you could paste the markdown in as a 'code block' so I can use it.

**Reply by tmorshed** [reply]
Hi! Yes of course, Ben. Will do that just now!
